starter-core/src/main/resources/pom.xml

```diff
@@ -12,6 +12,11 @@
         </repository>
     </repositories>
     <dependencies>
+        <dependency>
+            <groupId>com.agorapulse</groupId>
+            <artifactId>gru-micronaut</artifactId>
+            <version>1.0.1</version>
```

Comment:
I have release some minor patches this morning. one of them upgrading JSoup so Gru is no longer highlighted by IntelliJ due security vulnerability.

```suggestion
            <version>1.0.3</version>
```